src/main/java/seedu/duke/template/BinaryTree.java

```diff
@@ -7,17 +7,13 @@
 
 public class BinaryTree<T> {
     private final ArrayList<T> arrayList = new ArrayList<>(Collections.nCopies(16, null));
-    private final int spaces = 4;
+    private static final int SPACES_OF_FOUR = 4;
```

Comment:
Hmm consider naming it SPACES? Since we could change the number of spaces if required later on without having the name become obsolete

Reply:
Sure, will change it to SPACES.